gradle/installer-json.gradle

```diff
@@ -1,7 +1,10 @@
 import groovy.json.JsonOutput
 
 tasks.register("generateInstallerJson", GenerateInstallerJsonTask) {
-	configurations = [ common: "installer" ]
+	configurations = [ common: "installer", development: "development"]
+	repositoryOverrides = [
+	    "mixinextras-fabric": "https://repo.maven.apache.org/maven2/"
```

Comment:
Nit: Gradle has a constant for this URL: `ArtifactRepositoryContainer.MAVEN_CENTRAL_URL`.